Source/ACE/Entity/Landblock.cs

```diff
@@ -729,6 +733,13 @@ public void ScheduleItemTransfer(ActionChain chain, ObjectGuid wo, ObjectGuid co
             }
         }
 
+        public bool ItemExistsInWorld(ObjectGuid wo)
```

Comment:
I added that - when you get a message to put something in pack Put Item In Container from the client F7B1 0019 - you get 3 bits of data.   Item Guid, Container Guild and Location (this is overloaded) if the item is a container (or a foci)  going into a container, then location is the side slots for your character.   The determination of the item - container, foci or non container - is needed for our response in the F7B0 0022 where we send back 4 bits of information, the same 3 plus that identification of the item that is needed by the client.   It is called is_item_container and is always 0, 1 or 2 - 0 non container, 1 container and 2 foci.   I have created a property on world object to classify each world object as one of the 3.   

That was a long explanation, but basically when I get the message to put something in a pack, I have no way to know am I going internal pack to pack, from wielded to pack, pack to same pack different slot or ground to main pack, ground to slot in a side pack.   (another player or vendor handing us something is the same but without the ability to specify location, that action always uses slot 0)   So - why did I make that function?   To try and see which scenario I am dealing with.   I can kill that and just do the reverse.   I created a pack aware player inventory search so if I don't have it it must be from the world.   However, I have not seen a specific pcap of a player being handed something.   If it is the same message, that would be an issue for my current understanding on how this works.   I just made a post requesting a pcap.